packages/angular_devkit/architect/node/node-modules-architect-host.ts

```diff
@@ -109,4 +122,17 @@ export class WorkspaceNodeModulesArchitectHost implements ArchitectHost<NodeModu
 
     throw new Error('Builder is not a builder');
   }
+
+  private findProjectTarget(target: Target) {
+    if ('getProjectTargets' in this._workspace) {
```

Comment:
could you add a comment here? I'm guessing this conditional is for backwards compat with the old workspaces API?